runtime/include/chpl-gpu-impl.h

```diff
@@ -74,6 +74,7 @@ bool chpl_gpu_impl_stream_ready(void* stream);
 void chpl_gpu_impl_stream_synchronize(void* stream);
 
 bool chpl_gpu_impl_can_reduce(void);
+bool chpl_gpu_impl_can_cub_sort(void);
```

Comment:
again, let's not have "cub" in the interface. (similar to what we already have for reduce)